Table with a single row causes a panic.

The following line is giving the library problems:

```org
| Row |
````

Output (running inside hugo):

```
panic: runtime error: index out of range

goroutine 1283 [running]:
github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous.(*parser).generateTable(0xc4205d9600, 0xc4201976c0, 0xc4202de840, 0x10, 0x2b9)
	/Users/bep/go/src/github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous/goorgeous.go:395 +0x901
github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous.OrgOptions(0xc4205d8d00, 0x8ec, 0x900, 0xec62a0, 0xc4201e1e00, 0x0, 0x0, 0x0)
	/Users/bep/go/src/github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous/goorgeous.go:101 +0x6ab
github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous.Org(0xc4205d8d00, 0x8ec, 0x900, 0xec62a0, 0xc4201e1e00, 0xec2700, 0xc4201d5080, 0x100)
	/Users/bep/go/src/github.com/spf13/hugo/vendor/github.com/chaseadamsio/goorgeous/goorgeous.go:50 +0x53
github.com/spf13/hugo/helpers.orgRender(0xc4203d2ea0, 0xc420179ce0, 0x0, 0x0, 0x0, 0x0, 0xec2700, 0xc4201d5080, 0x40f928, 0x90, ...)
	/Users/bep/go/src/github.com/spf13/hugo/helpers/content.go:683 +0x1de
github.com/spf13/hugo/helpers.ContentSpec.RenderBytes(0xc420179ce0, 0x0, 0x0, 0x0, 0x0, 0xec2700, 0xc4201d5080, 0xc4203d2ea0, 0xc4201e17e0, 0x1, ...)
	/Users/bep/go/src/github.com/spf13/hugo/helpers/content.go:429 +0x15e
github.com/spf13/hugo/hugolib.(*Page).renderContent(0xc4204a2a00, 0xc4205d8400, 0x8ec, 0x900, 0xc4205d8400, 0x8ec, 0x900)
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/page.go:605 +0x28d
github.com/spf13/hugo/hugolib.commonConvert(0xc4204a2a00, 0xc42052e658, 0x9f8e61, 0xa97960, 0xc420192e10)
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/handler_page.go:140 +0x1ce
github.com/spf13/hugo/hugolib.orgHandler.PageConvert(0x0, 0x0, 0x0, 0xc4204a2a00, 0x99d668, 0xc42059459d, 0x3, 0xb70cc1)
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/handler_page.go:120 +0x2b
github.com/spf13/hugo/hugolib.(*orgHandler).PageConvert(0xc4201e0200, 0xc4204a2a00, 0xc4201e0200, 0xc42052e708, 0x99d548, 0xec0260)
	<autogenerated>:542 +0x69
github.com/spf13/hugo/hugolib.(*MetaHandle).Convert(0xc4201e1760, 0xb6db40, 0xc4204a2a00, 0xc4201b6240, 0xc4205972c0)
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/handler_meta.go:77 +0xa7
github.com/spf13/hugo/hugolib.pageConverter(0xc420597320, 0xc4205972c0, 0xc420192130)
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/site.go:1354 +0xfa
created by github.com/spf13/hugo/hugolib.(*Site).reProcess
	/Users/bep/go/src/github.com/spf13/hugo/hugolib/site.go:728 +0x14e0
```

Everything works fine if there is more than one row, but a single row is causing a panic.

I was able to reproduce the issue by adding the following to `TestRenderingTables`:

```
"table-single-cell": {
	"| r |\n",
	"\n<table>\n<tbody>\n<tr>\n<td>r</td>\n</tr>\n</tbody>\n</table>\n",
},
```
This produced the following output for `go test`:

```
--- FAIL: TestRenderingTables (0.00s)
panic: runtime error: index out of range [recovered]
	panic: runtime error: index out of range

goroutine 18 [running]:
testing.tRunner.func1(0xc420073c70)
	/usr/lib/go/src/testing/testing.go:622 +0x29d
panic(0x536a60, 0x602a10)
	/usr/lib/go/src/runtime/panic.go:489 +0x2cf
github.com/chaseadamsio/goorgeous.(*parser).generateTable(0xc420186d00, 0xc420180bd0, 0xc420180c60, 0x6, 0x40)
	/home/carl/programs/go/src/github.com/chaseadamsio/goorgeous/goorgeous.go:406 +0x901
github.com/chaseadamsio/goorgeous.OrgOptions(0xc42000feb0, 0x6, 0x8, 0x5f7500, 0xc42016cf00, 0x5f7500, 0xc42016cf00, 0xc42000feb0)
	/home/carl/programs/go/src/github.com/chaseadamsio/goorgeous/goorgeous.go:110 +0x6c3
github.com/chaseadamsio/goorgeous.OrgCommon(0xc42000feb0, 0x6, 0x8, 0xc42000feb0, 0x6, 0x8)
	/home/carl/programs/go/src/github.com/chaseadamsio/goorgeous/goorgeous.go:45 +0x88
github.com/chaseadamsio/goorgeous.testOrgCommon(0xc4201470e0, 0xc420073c70)
	/home/carl/programs/go/src/github.com/chaseadamsio/goorgeous/goorgeous_test.go:504 +0x120
github.com/chaseadamsio/goorgeous.TestRenderingTables(0xc420073c70)
	/home/carl/programs/go/src/github.com/chaseadamsio/goorgeous/goorgeous_test.go:450 +0x2c7
testing.tRunner(0xc420073c70, 0x569810)
	/usr/lib/go/src/testing/testing.go:657 +0x96
created by testing.(*T).Run
	/usr/lib/go/src/testing/testing.go:697 +0x2ca
exit status 2
FAIL	github.com/chaseadamsio/goorgeous	0.004s
```

--- a/goorgeous.go
+++ b/goorgeous.go
@@ -403,7 +403,10 @@ func isTable(data []byte) bool {
 func (p *parser) generateTable(output *bytes.Buffer, data []byte) {
 	var table bytes.Buffer
 	rows := bytes.Split(bytes.Trim(data, "\n"), []byte("\n"))
-	hasTableHeaders := reTableHeaders.Match(rows[1])
+	hasTableHeaders := len(rows) > 1
+	if len(rows) > 1 {
+		hasTableHeaders = reTableHeaders.Match(rows[1])
+	}
 	tbodySet := false
 
 	for idx, row := range rows {

--- a/goorgeous_test.go
+++ b/goorgeous_test.go
@@ -348,20 +348,20 @@ func TestRenderingFootnotes(t *testing.T) {
 	testCases := testCase{"Test 1[fn:1] and Test 2[fn: 2] and Test 3[fn:3] also test lettres[fn:let] then final test[fn:4]\n\n[fn:let] what?\n\n\n[fn:6] And test it[fn:7].\n\n* Footnotes\n\n[fn:1] Test 1\n\n[fn:3] Test 3\n\n[fn:2] Test2\n\n[fn:5] missing?\n\n[fn:6] Six?\n\n[fn:7] Seven??",
 		"<p>Test 1<sup class=\"footnote-ref\" id=\"fnref:1\"><a rel=\"footnote\" href=\"#fn:1\">1</a></sup> and Test 2[fn: 2] and Test 3<sup class=\"footnote-ref\" id=\"fnref:3\"><a rel=\"footnote\" href=\"#fn:3\">2</a></sup> also test lettres<sup class=\"footnote-ref\" id=\"fnref:let\"><a rel=\"footnote\" href=\"#fn:let\">3</a></sup> then final test<sup class=\"footnote-ref\" id=\"fnref:4\"><a rel=\"footnote\" href=\"#fn:4\">4</a></sup></p>\n\n<h1 id=\"footnotes\">Footnotes</h1>\n<div class=\"footnotes\">\n\n<hr />\n\n<ol>\n<li id=\"fn:1\">Test 1 <a class=\"footnote-return\" href=\"#fnref:1\"><sup>↩</sup></a></li>\n\n<li id=\"fn:3\">Test 3 <a class=\"footnote-return\" href=\"#fnref:3\"><sup>↩</sup></a></li>\n\n<li id=\"fn:let\">what? <a class=\"footnote-return\" href=\"#fnref:let\"><sup>↩</sup></a></li>\n\n<li id=\"fn:4\">DEFINITION NOT FOUND <a class=\"footnote-return\" href=\"#fnref:4\"><sup>↩</sup></a></li>\n</ol>\n</div>\n"}
 
-	flags := blackfriday.HTML_USE_XHTML;
+	flags := blackfriday.HTML_USE_XHTML
 	flags |= blackfriday.LIST_ITEM_BEGINNING_OF_LIST
 	flags |= blackfriday.HTML_FOOTNOTE_RETURN_LINKS
 
 	var parameters blackfriday.HtmlRendererParameters
 	parameters.FootnoteReturnLinkContents = "<sup>↩</sup>"
-	
-	renderer := blackfriday.HtmlRendererWithParameters(flags, "", "", parameters)
-	out := Org([]byte(testCases.in),renderer)
 
-	if !bytes.Equal(out,[]byte(testCases.expected)) {
-		t.Errorf("Footnote for Org() from: \n %s \n result: \n  %s\n wants:\n  %s",testCases.in, string(out), testCases.expected)
+	renderer := blackfriday.HtmlRendererWithParameters(flags, "", "", parameters)
+	out := Org([]byte(testCases.in), renderer)
+
+	if !bytes.Equal(out, []byte(testCases.expected)) {
+		t.Errorf("Footnote for Org() from: \n %s \n result: \n  %s\n wants:\n  %s", testCases.in, string(out), testCases.expected)
 	}
-}	
+}
 
 func TestRenderingBlock(t *testing.T) {
 
@@ -440,6 +440,10 @@ func TestRenderingTables(t *testing.T) {
 		"table-with-inlined-elements": {
 			"| Format           | Org mode markup syntax |\n| *Bold*           | =*Bold*=               |\n| /Italics/        | =/Italics/=            |\n| _Underline_      | =_Underline_=          |\n| =Verbatim=       | ==Verbatim== |\n| +Strike-through+ | =+Strike-through+=     |\n",
 			"\n<table>\n<tbody>\n<tr>\n<td>Format</td>\n<td>Org mode markup syntax</td>\n</tr>\n\n<tr>\n<td><strong>Bold</strong></td>\n<td><code>*Bold*</code></td>\n</tr>\n\n<tr>\n<td><em>Italics</em></td>\n<td><code>/Italics/</code></td>\n</tr>\n\n<tr>\n<td><span style=\"text-decoration: underline;\">Underline</span></td>\n<td><code>_Underline_</code></td>\n</tr>\n\n<tr>\n<td><code>Verbatim</code></td>\n<td><code>=Verbatim=</code></td>\n</tr>\n\n<tr>\n<td><del>Strike-through</del></td>\n<td><code>+Strike-through+</code></td>\n</tr>\n</tbody>\n</table>\n",
+		},
+		"table-single-cell": {
+			"| r |\n",
+			"\n<table>\n<tbody>\n<tr>\n<td>r</td>\n</tr>\n</tbody>\n</table>\n",
 		},
 	}
 

